lambda/main.go

```diff
@@ -61,7 +63,14 @@ func HandleRequest(ctx context.Context) error {
 		log.Printf("error while uploading exploit db file to s3. Bucket:%s, Region:%s,Error:%s", AWS_S3_BUCKET, AWS_S3_REGION, err)
 	}
 
-	return err
+	// Invalidate CloudFront cache for the uploaded file
+	err = invalidateCloudFrontCache(session)
```

Comment:
We should not invalidate the cache if there is some error while uploading file to s3 right?

If yes, we can return from

```
	if err != nil {
		log.Printf("error while uploading exploit db file to s3. Bucket:%s, Region:%s,Error:%s", AWS_S3_BUCKET, AWS_S3_REGION, err)
	}
```

Reply:
updated @namandf